src/index.ts

```diff
@@ -209,4 +210,49 @@ app.post("/attendance", async (c) => {
   return new Response();
 });
 
+app.get("/attendances/:course_id", async(c) => {
+  const subjectID = c.req.param('course_id');
+  const honoSession = c.get('session');
+  const session = honoSession.get("login") as Session | null;
+  if ( !session ){
+    return c.text("Unauthorized", 401);
+  }
+
+  //科目が存在するか、履修しているかをチェック
+  const [subjectResult, registration] = c.env.DB.batch([
+    c.env.DB.prepare("SELECT id FROM subjet WHERE id = ?").bind(subjectID),
+    c.env.DB.prepare("SELECT * registration WHERE subject_id = ?1 AND student_id = ?2").bind(subjectID, session.account.id)
+  ])
+  const subjectExists = subjectResult.results.length === 1;
+  const isRegistration = registration.results.length === 1;
+  if ( !subjectExists ){
+    return c.text("Not Found", 404);
+  }
+  if ( !isRegistration ){
+    return c.text("Not Found", 404);
+  }
+
+  /*出席状況を取得*/
+  const { on_time, late, miss } = c.env.DB
+  .prepare(`
+    SELECT
+      SUM(CASE WHEN attendance.created_at BETWEEN attendance_board.start_from AND (atendance.created_at + attendance_board.seconds_from_start_to_be_late) THEN 1 ELSE 0 END) AS on_time,
+      SUM(CASE WHEN attendance.created_at BETWEEN (attendance.created_at + attendance_board.seconds_from_start_to_be_late) AND (attendance.created_at + attendance_board.seconds_from_start_to_be_late + attendance_board.seconds_from_be_late_to_end) THEN 1 ELSE 0 END) AS late,
+      SUM(CASE WHEN attendance.created_at >= (attendance.created_at + attendance_board.seconds_from_start_to_be_late + attendance_board.from_be_late_to_end) THEN 1 ELSE 0 END) AS miss
+    FROM attendance
+    INNER JOIN attendance_board
+      ON attendance.\"where\" = attendance.board.id
```

Comment:
```suggestion
      ON attendance."where" = attendance_board.id
```
これはバッククォート `` ` `` で囲ったテンプレートリテラルの中なので, `"` はエスケープせずに書けると思います. またカラムは `attendance_board` の `id` だと思います.